docs/user_guide/docs/mapl_code_generator.md

```diff
@@ -1,27 +1,30 @@
 ## MAPL Automatic Code Generator
 
 Any ESMF gridded component typically requires an Import State and an Export State (if necessary an Internal State too).
-Each of the states contains member variables (Fields, Bundles) that need to be registered  before they are used.
+Each of the states contains member variables (Fields, Bundles) that need to be registered before they are used.
 The number of the those variables can be large and make the declaration process cumbersome
 (possibly missing fields) and the declaration section in the code extremely long.
 
-MAPL has a utility tool (named [MAPL_GridCompSpecs_ACG.py
+MAPL has a code generator utility tool (named [MAPL_GridCompSpecs_ACG.py
```

Comment:
Maybe it is just me, but this document should lead in with a sentence (or paragraph) about what it is.     And only then jump into why you might want to use it.